test_files/jsdoc/jsdoc.ts

```diff
@@ -115,4 +115,15 @@ const somePolymerBehavior = {};
  * if they are declared via the Polymer function.
  */
 let Polymer: any;
-Polymer({ behaviors: [ 'test' as any ] });
+Polymer({behaviors: ['test' as any]});
+
+/**
+ * This class has a 'template' tag, which we want to allow (because this is
+ * how to doc this) but not let Closure interpret (because we emit our own).
+ * The desired behavior is that the user-written @template comment (which
+ * talks about T) is dropped, but the tsickle-generated @template comment
+ * (which talks about T2) is preserved.
+ *
+ * @template T User-written comments on the template.
+ */
+class Foo<T2> {}
```

Comment:
Should we also have a test for the proper usage of this "feature", i.e. matching TS and doc template name.

Reply:
Done.